Buildscripts/release-summary.py

```
@@ -0,0 +1,84 @@
+        "If a pull request or commit has multiple changes add them as separate entries."
+        "Ignore irrelevant changes: documentation, minor fixes, non-descript fixes, scripts, github workflows."
+        "For 'New Devices': only mention the device name, don't like pull request."
+        "For 'Improvements & Fixes': Avoid non-descript entries, write specifics."
+        "For 'TactilitySDK': Avoid generic changes, be specific. Mention API changes."
+    )
+
+
+def run_claude(commits_md: str, prs_md: str) -> str:
+    prompt = build_prompt()
+    stdin_content = f"## Commits\n\n{commits_md}\n\n## Pull Requests\n\n{prs_md}"
+    result = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)